src/scripts/generate_binary_ops.py

```diff
@@ -0,0 +1,317 @@
+        return {'oper': '<<', 'op2': op2, 'sum': sum}
+
+    # Bitshifting to right
+    # Python only can handle shiftcount between 0-31, but smaller and bigger numbers
+    # are also can be simulated with this method
+    def bitrightshift(self, op1, op2):
+        op2 = random.randint(_constants.bitmin, _constants.bitmax)
+        sum = int(hex((op1 >> ((op2 % (_constants.bitmax_exposant + 1)) & 0x1f))), 16)
+        return {'oper': '>>', 'op2': op2, 'sum': sum}
+
+    # Unsigned right bitshifting
+    def bitunsignedrightshift(self, op1, op2):
+        op2 = random.randint(_constants.uint32min, _constants.uint32max)
+        lnum = unsigned32(op1)
+        rnum = unsigned32(op2)
+        shiftcount = rnum & 0x1f
+        sum = ((lnum >> shiftcount) % _constants.uint32max)
+        return {'oper': '>>>', 'op2': op2, 'sum': sum}
+
+    # Bitwise and
+    def bitand(self, op1, op2):
+        sum = op1 & op2
+        return {'oper': '&', 'op2': op2, 'sum': sum}
+
+    # Bitwise or
+    def bitor(self, op1, op2):
+        sum = op1 | op2
+        return {'oper': '|', 'op2': op2, 'sum': sum}
+
+    # Bitwise xor (exclusive or)
+    def bitxor(self, op1, op2):
+        sum = op2 ^ op1
+        return {'oper': '^', 'op2': op2, 'sum': sum}
+
+    # Setting maximum and minimum depending on operation type
+    def set_min_max(self, calc):
+        if calc == self._InnerValues.arith:
+            MINNUM = _constants.min
+            MAXNUM = _constants.max
+        elif calc == self._InnerValues.bit:
+            MINNUM = _constants.bitmin
+            MAXNUM = _constants.bitmax
+        else:
+            MINNUM = _constants.uint32min
+            MAXNUM = _constants.uint32max
+        return {'MIN' : MINNUM, 'MAX': MAXNUM}
+
+    def create_declarations(self, expected_value, MINNUM, MAXNUM):
+        # Generating false numbers
+        false_numbers = self.generate_false_numbers(expected_value, self._InnerValues.false_values_count, MINNUM,
+        MAXNUM)
+        # Creating and adding false numbers string
+        false_numbers_string = self.create_false_numbers_array_string(self._InnerValues.false_values_count)
+        # Creating and adding declarations of test values
+        test_case = self.create_declarations_string(self.create_declarations_dict(self.operands,
+        self.TEST_VALUE_VAR_NAME))
+        # Creating and adding declarations of false values
+        test_case += self.create_declarations_string(self.create_declarations_dict(false_numbers,
+        self.FALSE_RESULT_VAR_NAME))
+        # Creating and adding declarations of expected value
+        test_case += self.create_declarations_string(self.create_declarations_dict(expected_value,
+        self.EXPECTED_VAR_NAME))
+        return {'test_case' : test_case, 'false_numbers_string' : false_numbers_string}
+
+    # Converting expression, expected value, false value in a function call to string format
+    def string_creator(self, MINNUM, MAXNUM):
```

Comment:
How about using the result object of `set_min_max` function instead of using it's properties separately?